TYPO3.Neos/Resources/Public/JavaScript/Content/Model/Node.js

```diff
@@ -203,7 +203,7 @@ define([
 				label = (this.get('nodeTypeLabel') || this.get('nodeType')) + ' (' + this.getAttribute('_name') + ')';
 			}
 
-			label = $('<span>' + label + '</span>').text().trim();
+			label = $('<u/>').html(label).text().trim();
```

Comment:
I see! The reason why i asked was because i thought  `label = $(label).text().trim();`would do just the same, but that's not true since it would return an empty string if the label wasn't wrapped in a tag (because of `text()`)